platform/core/src/poly_out.rs

```diff
@@ -0,0 +1,112 @@
+use std::fs::DirBuilder;
+use std::path::Path;
+use uuid::Uuid;
+
+type ResourceId = String;
+type FsId = String;
+
+struct Metadata {}
+struct Content {}
+
+trait PolyOutTrait {
+    fn metadata(resource_id: ResourceId) -> Result<Metadata, String>;
+    fn read(resource_id: ResourceId) -> Result<Content, String>;
+    fn remove(resource_id: ResourceId) -> Result<(), String>;
+    // import will decide if it needs to unzip a file. The contents of the url will be extracted and inserted into a database.
+    fn import(
+        url: String,
+        feature_name: String,
```

Comment:
That should go - the file system is local to features by design, so it shouldn't be possible for one feature to import files into another one.

Reply:
Done